staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -95,6 +95,9 @@ type ExtraConfig struct {
 	// KCP
 	ClusterAwareCRDLister  kcp.ClusterAwareCRDClusterLister
 	TableConverterProvider TableConverterProvider
+	// NoSSA deactivates SSA for a specific API server instead of globally through the feature gate
+	// used with the embedded cache server in kcp
+	NoSSA bool
```

Comment:
I would prefer to spell this out - something like `DisableServerSideApply`. Although double negatives can be difficult to reason about - `if !DisableServerSideApply { // it's enabled }` ... but I realize we want to enable by default...

Reply:
I changed the name accordingly.